client/app/vps/modal/shortcut/password/vps-password.controller.js

```diff
@@ -0,0 +1,49 @@
+class VpsPasswordCtrl {
+    constructor ($translate, $uibModalInstance, CloudMessage, serviceName, VpsService) {
+        this.$translate = $translate;
+        this.$uibModalInstance = $uibModalInstance;
+        this.CloudMessage = CloudMessage;
+        this.serviceName = serviceName;
+        this.VpsService = VpsService;
+
+        this.loader = {
+            init: false,
+            save: false
+        };
+        this.model = {};
+        this.selected = {
+            rescue: true
+        };
+    }
+
+    $onInit () {
+        this.loader.init = true;
+        this.VpsService.getTaskInError(this.serviceName)
+            .then(tasks => { this.hasTasks(tasks) })
+            .catch(err => this.CloudMessage.error(err))
+            .finally(() => { this.loader.init = false });
+    }
+
+    hasTasks (tasks) {
```

Comment:
This return nothing, should be private and display an error message despite being an exception 

And it's not old code, it's an addon....

Reply:
Yes I have done that part of code.